### PR TITLE
Updated with new file name for Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by ImakuTranslate**
For testing purpose, using zh_Hans as file name

### CODE REVIEW

File: locale/zh_Hans.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        tenantid: 租户ID
+        username: 用户名
+        password: 密码
+        region: 地区
+    buttons:
+      save: 保存设定
+    help_texts:
+      description: '设置上传服务与偏好设定
+
+        '
```
Comment (Luceos) [inline locale/zh_Hans.yml]: What are these? Seems your files are badly formatted..

**Comment by tony199555** [comment]
I think the format problem is cause by using Weblate and I am following the format it showed me. Anyway, cn.yml and format is fixed though I don't have a local testing machine now for testing so I don't know if it will work.

**Comment by Luceos** [comment]
Hm, might need to drop that Hans file..

**Comment by tony199555** [comment]
Right, after talking on forum and with few other translator I found out that the code that will determines what language flarum should use is actually in the composer.json, under "flarum-locale". That which means no easy way to do it unless our lead devs decide to enforce more unity. As far as I concern, I might just do a full translation by my own. So feel free to do whatever you want ( change to zh.yml or something else).

**Pull request merged**